api/auth/center_test.go

```diff
@@ -432,7 +433,7 @@ func TestAwsEncodedWithRequest(t *testing.T) {
 func chunkSlice(payload []byte, chunkSize int) [][]byte {
 	var result [][]byte
 
-	for i := 0; i < len(payload); i += chunkSize {
+	for i := range len(payload) {
```

Comment:
This doesn't seem to be correct, we were iterating with `+= chunkSize` steps.

Reply:
Damn, updated